tracer/test/test-applications/integrations/Samples.InstrumentedTests/InstrumentationTestsBase.cs

```diff
@@ -418,7 +418,21 @@ public static T TestRealDDBBLocalCall<T>(Func<T> expression)
         }
         else
         {
-            return expression.Invoke();
+            try
+            {
+                return expression.Invoke();
+            }
+            catch (System.Data.SqlClient.SqlException ex) // System.Data.SqlClient.SqlException : Timeout expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.
+            {
+                if (ex.Message.StartsWith("Timeout expired", StringComparison.OrdinalIgnoreCase))
+                {
+                    return default(T);
+                }
+                else
+                {
+                    throw;
+                }
+            }
```

Comment:
We can simplify this by using `when` 🙂 
```suggestion
            catch (System.Data.SqlClient.SqlException ex) when (ex.Message.StartsWith("Timeout expired", StringComparison.OrdinalIgnoreCase))
            {
                // System.Data.SqlClient.SqlException : Timeout expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.
                return default(T);
            }
```